src/EPR.CommonDataService.Api/Controllers/SubmissionsController.cs

```diff
@@ -1,28 +1,34 @@
+#nullable disable
```

Comment:
why have you disabled nullability? Its enabled for a reason!

Reply:
We have new injections of Logger and Config.
it's complaining about possible null value for config.

There are no alerts anywhere in the code

Reply:
removed